.github/workflows/ci.yml

```diff
@@ -153,8 +153,39 @@ jobs:
               --no-output \
               +${{ matrix.test }}
 
+  acir-bench:
+    runs-on: ubuntu-latest
+    needs: setup
```

Comment:
Should this come after `build` so we make sure that bb is built before getting here, and we don't have two jobs potentially trying to run it at the same time? Not sure if that's how we should be structuring the CI tbh.

Reply:
Isn't this just handled by earthly? If `acir-bench` starts building `bb` first then `build` will build up until `bb`, use the cached result from `acir-bench` and then continue from there.

Reply:
Tbh I don't know whether one build will "block" the others so they wait until it's ready, or they will both run if the first build isn't completed.